test/render_smf_mem.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Renders a Standard MIDI File (SMF) to an audio file using a given SF2 soundfont."""
+
+import sys
+from os.path import splitext
+from fluidsynth import Synth, Player
+
+
+def main(args=None):
+    if len(args) < 2:
+        return "Usage: render_smf.py <SF2 file> <SMF input file> [<audio output filename>]"
```

Comment:
The usage string is incorrect, since the file is called render_smf_mem.py. Since it's virtually identical to render_smf.py, I'd suggest merging them into one and making the file loading operation an optional parameter